### PR TITLE
added multibot explanation

### PR DESCRIPTION
**Pull request by Cypherke**
**Changes made: added multibot configuration**

### CODE REVIEW

File: multibotconfig.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  "spawnRadius" : -1,
+  "banSpinCount" : 0,
+  "transferCpThreshold" : 400,
+  "transferIvThreshold" : 80,
+  "ignoredPokemon" : [ "EEVEE", "MEWTWO", "CHARMANDER" ],
+  "obligatoryTransfer" : [ "DODUO", "RATTATA", "CATERPIE", "PIDGEY" ],
+  "export" : "",
+  "guiPortSocket" : 8001,
+  "initialMapSize" : 3,
+  "version" : "0.5.0-alpha4"
```
Comment (jabbink) [inline multibotconfig.json]: should be removed I guess, as this is should be generated at runtime. should probably also remove it in the `data class Settings` annotation. @apottere ?

**Comment by konstantinj** [comment]
Hi,
in my opinion it's too much maintenance to keep this working with future versions. It's quite easy to run multiple bots using docker containers. If you like to use the gui you can just open different ports on your host machine.

File: multibotconfig.json (hunk 1 above)
Comment (apottere) [inline multibotconfig.json]: Yup, in `@JsonIgnoreProperties`

**Comment by apottere** [comment]
Why was this closed?

**Comment by Cypherke** [comment]
I m guessing it 'll still change some more in the coming days

**Pull request closed without merging**